foundry.toml

```diff
@@ -1,6 +1,21 @@
 [profile.default]
+solc_version = "0.8.19"
+optimizer = true
+optimizer_runs = 200
```

Comment:
has any gas profiling been done? Wondering it it's helpful to increase this, to say `1000`?

Reply:
No profiling has been done yet. This could be good to do once all components are code complete.